solr/core/src/java/org/apache/solr/search/CacheConfig.java

```diff
@@ -126,7 +136,9 @@ public static CacheConfig getConfig(SolrConfig solrConfig, String nodeName, Map<
     config.cacheImpl = config.args.get("class");
     if (config.cacheImpl == null) config.cacheImpl = "solr.CaffeineCache";
     config.regenImpl = config.args.get("regenerator");
-    config.clazz = loader.findClass(config.cacheImpl, SolrCache.class);
+    config.clazzName = config.cacheImpl;
```

Comment:
why clazzName here instead of cacheImplName or some such? just want to follow the naming scheme